bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -0,0 +1,155 @@
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opengarage.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.library.unit.MetricPrefix;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.openhab.binding.opengarage.internal.OpenGarageWebTargets;
+import org.openhab.binding.opengarage.internal.api.ControllerVariables;
+import org.openhab.binding.opengarage.internal.OpenGarageConfiguration;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * The {@link OpenGarageHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Paul Smedley - Initial contribution
+ */
+
+@NonNullByDefault
+public class OpenGarageHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(OpenGarageHandler.class);
+
+    private long refreshInterval;
+
+    private @Nullable OpenGarageWebTargets webTargets;
+    private @Nullable ScheduledFuture<?> pollFuture;
+
+    public OpenGarageHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        try {
+            handleCommandInternal(channelUID, command);
+        } catch (OpenGarageCommunicationException ex) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, ex.getMessage());
+        }
+    }
+
+    private void handleCommandInternal(ChannelUID channelUID, Command command) throws OpenGarageCommunicationException {
+        logger.warn("Received command {} for thing '{}' on channel {}", command, thing.getUID().getAsString(),
+                    channelUID.getId());
+        switch (channelUID.getId()) {
+            case OpenGarageBindingConstants.CHANNEL_OG_STATUS:
+                if (command instanceof OnOffType) {
+                    changeStatus(((OnOffType) command).equals(OnOffType.ON));
+                    return;
+                }
+                break;
+        }
```

Comment:
`switch`-statements should have a default branch. You don‘t need to do anything in there.

Reply:
Fixed in 1641769b62